lib/appsignal.ex

```diff
@@ -116,7 +116,9 @@ defmodule Appsignal do
 
   @doc false
   def add_default_probes do
-    # Stub for adding default probes
+    unless Mix.env() in [:test, :test_phoenix, :test_no_nif] do
```

Comment:
we don't want to test if this function does what it does?

We could move the `unless` -statement to wherever this function is called

Reply:
I'm usually more in favour of having this type of calls in a "Tell don't ask manor". I.e. I want to be able to call a method wherever, without having to think about what context I need to put around it. So if I were to make the decision I want to keep it here.

@jeffkreeftmeijer If we want to test to see if the `add_default_probes` is being called, what would be the way to do so in Elixir?

Reply:
We can't call out to `Mix.env/0` here, as it's not available in runtime in production releases. That's also why we wrap functions in these checks if we need to, to make them run in compile time.

Why is it a problem to call this method in the tests? I'm suspecting that's because the `FakeAppsignal` agent isn't running, which we could fix by not making the `FakeAppsignal` module dependent on the process running. I'm looking into how that could work and will report back.

Reply:
Interesting, I wonder why this actually works :/ The problem was indeed that FakeAppsignal was not running.

Reply:
We moved the Mix out of the method and created a better solution for this.

@jeffkreeftmeijer and I discussed testing this method, and we figure it is not needed for now.